Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor

```diff
@@ -1,6 +1,170 @@
 ﻿@page "/card";
+@using OpenCodeDev.Blazor.Foundation.Doc.Core.Components.Containers.Section;
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Card</h2>
+        <p>Cards are a popular and flexible UI component.</p>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Basics</h2>
+        <p>
+            A card is just an element with a <Code Content="PENhcmQ+"></Code> applied. You can put any kind of content inside. Make sure you wrap your content in a <Code Content="PENhcmRTZWN0aW9uPg=="></Code> element in order to achieve the traditional card look.
+        </p>
+        <p>
+            A card container has no padding, allowing you to place full-bleed images inside. Use the <Code Content="PENhcmREaXZpZGVyPg=="></Code> and <Code Content="PENhcmRTZWN0aW9uPg=="></Code> tags to sub-divide a card.
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PENhcmQgc3R5bGU9IndpZHRoOiAzMDBweDsiPgogIDxDYXJkRGl2aWRlcj4KICAgIFRoaXMgaXMgYSBoZWFkZXIKICA8L0NhcmREaXZpZGVyPgogIDxpbWcgc3JjPSJhc3NldHMvaW1nL2dlbmVyaWMvcmVjdGFuZ2xlLTEuanBnIj4KICA8Q2FyZFNlY3Rpb24+CiAgICA8aDQ+VGhpcyBpcyBhIGNhcmQuPC9oND4KICAgIDxwPkl0IGhhcyBhbiBlYXN5IHRvIG92ZXJyaWRlIHZpc3VhbCBzdHlsZSwgYW5kIGlzIGFwcHJvcHJpYXRlbHkgc3ViZHVlZC48L3A+CiAgPC9DYXJkU2VjdGlvbj4KPC9DYXJkPg==" />
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.Card style="width: 300px;">
+            <OpenCodeDev.Blazor.Foundation.Components.Containers.CardDivider>
+                This is a header
+            </OpenCodeDev.Blazor.Foundation.Components.Containers.CardDivider>
+            <img src="https://images.hdqwalls.com/download/saturn-planet-8k-3i-1336x768.jpg">
+            <OpenCodeDev.Blazor.Foundation.Components.Containers.CardSection>
+                <h4>This is a card.</h4>
+                <p>It has an easy to override visual style, and is appropriately subdued.</p>
+            </OpenCodeDev.Blazor.Foundation.Components.Containers.CardSection>
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.Card>
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.Callout TypeClass="alert">
```

Comment:
We do not support IE so notice must be removed

Reply:
I deleted a Callout
```suggestion
        <OpenCodeDev.Blazor.Foundation.Components.Containers.CardSection>
                <h4>This is a card.</h4>
                <p>It has an easy to override visual style, and is appropriately subdued.</p>
            </OpenCodeDev.Blazor.Foundation.Components.Containers.CardSection>
        </OpenCodeDev.Blazor.Foundation.Components.Containers.Card>
        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
```